CHANGELOG.md

```diff
@@ -1,94 +1,155 @@
-[unreleased]
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+Note that in line with [Django REST Framework policy](http://www.django-rest-framework.org/topics/release-notes/),
```

Comment:
One concern I have is that DRF does not follow semver:

> Medium version numbers (0.x.0) may include API changes, in line with the deprecation policy. You should read the release notes carefully before upgrading between medium point releases.

Where semver requires only new backwards-compatible changes.

As DJA is an extension of DRF, should we follow DRF's versioning or be more explicit that we instead follow semver?

Reply:
let's use semver which is commonly accepted.